### PR TITLE
Added sprites from artist into game and changed script.rpy to show cpp

### PR DESCRIPTION
**Pull request by AntonCSalvador**
Added the artist's demo images into the game with cpp being displayed in the game. Also changed the README for simple instructions for Linux bc it is the renpy.sh file.

### CODE REVIEW

**Comment by AntonCSalvador** [comment]
lgtm

**Review by WilsonGoins — COMMENTED** [review]
LGTM, waiting for ryder's approval

**Review by JMalegni — CHANGES_REQUESTED** [review]
WE are no longer using script.rpy as our main script files. Please look into the script directory and update accordingly.

**Pull request closed without merging**
